.travis.yml

```diff
@@ -31,7 +30,7 @@ script:
 
 after_script:
   - |
-    if [[ "$TRAVIS_PHP_VERSION" != 'hhvm' && "$TRAVIS_PHP_VERSION" != '7.0' ]]; then
+    if [[ "$TRAVIS_PHP_VERSION" != 'hhvm' ]]; then
```

Comment:
Shouldn't this be dropped, since we no longer have `hhvm` set? 

Btw, code coverage doesn't (yet) exist with PHP 7.3, so this line should read:

```yaml
if [[ -f coverage.clover ]]; then
```

In order to ensure that "no coverage" is sent to Scrutinizer.